runners/direct-java/src/main/java/org/apache/beam/runners/direct/ParDoMultiEvaluatorFactory.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Licensed to the Apache Software Foundation (ASF) under one
+* Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Apache header is enforced by `apache-rat-plugin`. I believe this was chosen to support more non-Java file formats.